modules/aws/aurora/examples/aurora-mysql/terraform.defaults

```diff
@@ -0,0 +1,33 @@
+default_tags = {
+    "argonaut.dev/name"        = "test"
+    "argonaut.dev/type"        = "Aurora"
+    "argonaut.dev/manager"     = "argonaut.dev"
+    "argonaut.dev/rds-engine"  = "aurora-mysql"
+    "argonaut.dev/env/dev" = "true"
```

Comment:
Change format to `"argonaut.dev/env" = "dev"`
Also add another line `"env" = "dev"` # this is a standard tag that many 3p tools read and use

@PrashantRaj18198 we need to do this for all the other pieces as well. Bad decision with the wrong format earlier - sorry 😞

Reply:
@suryaoruganti  Have added a separate ticket so it doesn't get skipped - https://linear.app/argonaut/issue/ARG-937/refactor-default-tags-for-all-existing-tf-modules